src/main/java/uk/gov/hmcts/reform/userprofileapi/domain/UpdateUserDetails.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.reform.userprofileapi.domain;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.apache.commons.lang.StringUtils;
+
+@Getter
+@Setter
+public class UpdateUserDetails {
+
+    protected String forename;
+    protected String surname;
+    protected Boolean active;
+
+
+    public UpdateUserDetails(String foreName, String surname, Boolean active) {
```

Comment:
keep the case consistent. Should be forename NOT foreName as per the member variables